pkg/reader/ccip.go

```diff
@@ -1397,10 +1397,6 @@ func (r *ccipChainReader) GetMedianDataAvailabilityGasConfig(
 		}
 	}
 
-	if len(overheadGasValues) == 0 {
-		return cciptypes.DataAvailabilityGasConfig{}, fmt.Errorf("no valid fee quoter destChainConfigs found")
-	}
-
```

Comment:
If this slice is empty, what are the returned values? All zeros? Will that break stuff downstream or nah?

Reply:
the rest of the flow only involve multiplication and addition. I don't think it will have an impact on it.

Basically:
```
totalCost = exec cost + da cost
da cost will be equal to 0
```